playbooks/roles/common/tasks/main.yml

```diff
@@ -62,3 +62,7 @@
   with_items:
     - { src: "20auto-upgrades.j2",       dest: "/etc/apt/apt.conf.d/20autoupgrades" }
     - { src: "50unattended-upgrades.j2", dest: "/etc/apt/apt.conf.d/50unattended-upgrades" }
+
+- name: Apply the sysctl values
```

Comment:
Nit: Maybe the `name` could be a little clearer as "Apply custom sysctl values"?

Reply:
Ended up removing this since it happens in the role anyway